docs/build_integration.md

```diff
@@ -73,6 +73,9 @@ $ cd /path/to/oss-fuzz
 $ export PATH_TO_PROJECT=<path_to_your_project>
 $ python infra/helper.py generate --external --language=c++ $PATH_TO_PROJECT
 ```
+Note that you may need to change the `--language` argument to another value
+if your project is written in another language, this is discussed more in the
```

Comment:
Just a grammar nit. Either change the comma after "language" to a period, or put the second clause in parentheses like this:

`Note that you may need to change the `--language` argument to another value if your project is written in another language (discussed more in the...)`